src/main/views/courts/tabs/openingHoursContent.njk

```diff
@@ -37,19 +37,27 @@
 
 {% for openingTime in opening_times %}
 
+  {% set isNew = (openingTime.isNew === true) or (openingTime.isNew === "true") %}
+
   {% set descriptionError = false %}
   {% if openingTime.type_id === '' %}
-    {% set descriptionError = "{ text: 'Type is required' }" %}
+    {% set descriptionError = { text: 'Type is required' } %}
```

Comment:
Is the wrapping with "" not necessary? Just wondering as I saw @Majid-justice add these onto one of his amendments for the tabs he has worked on. Unsure which is correct without diving in and testing myself.

Reply:
This change is correct. Currently, the error message is not displaying next to the input element and this fixes it.

Reply:
Awesome, cheers for clarifying @agraham01